native/react-native/common-components/error-handler.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react';
+import {Modal, Text, TouchableHighlight, View} from 'react-native';
+import PropTypes from "prop-types";
+
+/**
+ * 
+ * This component serves as a global error handler template.
+ * 
+ * @props:
+ *  - errorHeader: A string describing what kind of error is being thrown.
+ *  - errorInfo: A string describing the details of the error.
+ *  - modalVisible: A bool for assigning the initial value of the modal's visiblity
+ *  - onClose: A function that gets called when the modal closes.
+ */
+
+class ErrorHandler extends Component {
```

Comment:
This is incredibly useful.

Reply:
Now I'm gonna have to go in and make this thing actually look nice!